tests/system/python/api/test_browser_assets.py

```diff
@@ -0,0 +1,134 @@
+        conn = http.client.HTTPConnection(foglamp_url)
+        conn.request("GET", '/foglamp/asset')
+        r = conn.getresponse()
+        assert 200 == r.status
+        r = r.read().decode()
+        jdoc = json.loads(r)
+        assert len(jdoc), "No data found"
+        assert ASSET_NAME == jdoc[0]["assetCode"]
+        assert 1 == jdoc[0]["count"]
+
+    def test_get_asset(self, foglamp_url):
+        conn = http.client.HTTPConnection(foglamp_url)
+        conn.request("GET", '/foglamp/asset/{}'.format(ASSET_NAME))
+        r = conn.getresponse()
+        assert 200 == r.status
+        r = r.read().decode()
+        jdoc = json.loads(r)
+        assert len(jdoc), "No data found"
+        assert {'sensor': SENSOR_VALUE} == jdoc[0]['reading']
+
+    def test_get_asset_all_readings_summary(self, foglamp_url):
+        conn = http.client.HTTPConnection(foglamp_url)
+        conn.request("GET", '/foglamp/asset/{}/summary'.format(ASSET_NAME))
+        r = conn.getresponse()
+        assert 200 == r.status
+        r = r.read().decode()
+        jdoc = json.loads(r)
+        assert len(jdoc), "No data found"
+        summary = jdoc[0]['sensor']
+        assert SENSOR_VALUE == summary['average']
+        assert SENSOR_VALUE == summary['max']
+        assert SENSOR_VALUE == summary['min']
+
+    def test_get_asset_reading(self, foglamp_url):
+        conn = http.client.HTTPConnection(foglamp_url)
+        conn.request("GET", '/foglamp/asset/{}/sensor'.format(ASSET_NAME))
+        r = conn.getresponse()
+        assert 200 == r.status
+        r = r.read().decode()
+        jdoc = json.loads(r)
+        assert len(jdoc), "No data found"
+        assert SENSOR_VALUE == jdoc[0]['sensor']
+
+    def test_get_asset_summary(self, foglamp_url):
+        conn = http.client.HTTPConnection(foglamp_url)
+        conn.request("GET", '/foglamp/asset/{}/sensor/summary'.format(ASSET_NAME))
+        r = conn.getresponse()
+        assert 200 == r.status
+        r = r.read().decode()
+        jdoc = json.loads(r)
+        assert len(jdoc), "No data found"
+        summary = jdoc['sensor']
+        assert SENSOR_VALUE == summary['average']
+        assert SENSOR_VALUE == summary['max']
+        assert SENSOR_VALUE == summary['min']
+
+    def test_get_asset_averages(self, foglamp_url):
+        conn = http.client.HTTPConnection(foglamp_url)
+        conn.request("GET", '/foglamp/asset/{}/sensor/series'.format(ASSET_NAME))
+        r = conn.getresponse()
+        assert 200 == r.status
+        r = r.read().decode()
+        jdoc = json.loads(r)
+        assert len(jdoc), "No data found"
+        assert SENSOR_VALUE == jdoc[0]['average']
+        assert SENSOR_VALUE == jdoc[0]['max']
+        assert SENSOR_VALUE == jdoc[0]['min']
+
+    # TODO: Add more paramterized tests with additional params like limit, skip, group, timeunits etc.
+    # And supply more data (nested readings) instead of 1 single -level reading
```

Comment:
yes, let's supply atleast 3 values e.g. 1, 2, 0 and verify avg 1, max 2 and min 0. if we can use int and float both, that will be more valuable.